pkg/search/postgres/select.go

```diff
@@ -39,6 +39,7 @@ func RunSelectRequestForSchema[T any](ctx context.Context, db postgres.DB, schem
 	}()
 
 	query, err = standardizeSelectQueryAndPopulatePath(ctx, q, schema, SELECT)
+	fmt.Println(query.AsSQL())
```

Comment:
Not that it really matters in the scheme of life, but this is where I log the query.
https://github.com/stackrox/stackrox/blob/78463b3641c8d863787869dc3c111a6ef00e88ca/pkg/search/postgres/common.go#L306